src/components/WelcomeModal/WelcomeModal.stories.tsx

```diff
@@ -1,3 +1,5 @@
+import { MockedProvider } from "@apollo/client/testing";
+import { StoryObj } from "@storybook/react";
 import {
```

Comment:
I'm seeing that the modals persist on the docs page. I googled it and it's a problem that others have run into, but I couldn't get their solutions to work 🤔 May be worth opening another ticket if it's not easy to fix!

<img width="1461" alt="Screenshot 2023-03-23 at 7 56 24 PM" src="https://user-images.githubusercontent.com/47064971/227395545-74ee269f-c7a6-4fb6-a18a-1feefe46e988.png">